File: src/features/standings/models/standingRow.ts
import { Type } from '@sinclair/typebox';

export const StandingRow = Type.Object({
  position: Type.Integer(),
  teamId: Type.Integer(),
  teamName: Type.String(),
  played: Type.Integer(),
  won: Type.Integer(),
  draw: Type.Integer(),
  lost: Type.Integer(),
  goalsFor: Type.Integer(),
  goalsAgainst: Type.Integer(),
  goalDifference: Type.Integer(),
  points: Type.Integer(),
  form: Type.Union([Type.String(), Type.Null()]),
});

File: true/Models/StandingRow.swift
import Foundation

struct StandingRow: Codable {
    let position: Int
    let teamId: Int
    let teamName: String
    let played: Int
    let won: Int
    let draw: Int
    let lost: Int
    let goalsFor: Int
    let goalsAgainst: Int
    let goalDifference: Int
    let points: Int
    let form: String?
}
